instrumentation/google.golang.org/grpc/otelgrpc/interceptor.go

```diff
@@ -347,11 +363,17 @@ func UnaryServerInterceptor(opts ...Option) grpc.UnaryServerInterceptor {
 			statusCode, msg := serverStatus(s)
 			span.SetStatus(statusCode, msg)
 			span.SetAttributes(statusCodeAttr(s.Code()))
-			messageSent.Event(ctx, 1, s.Proto())
+
```

Comment:
This meaningless blank line will not increase the readability of code, it could be removed.
Ditto with L373 L404 L417.

Reply:
Fixed.